Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/transaction/DccJWK.kt

```diff
@@ -6,10 +6,32 @@ import kotlinx.parcelize.Parcelize
 
 @Parcelize
 data class DccJWK(
-    val x5c: List<String>, // base64-encoded list of strings that represents an x509 certificate
-    val kid: String, // base64-encoded string
-    val alg: String, // string describing the algorithm
-    val use: Purpose,
+    /**
+     * a base64-encoded string that represents an x509 certificate.
+     *
+     * The attribute typically needs to be parsed to a native x509 certificate object
+     * to obtain certificate information such as the included public key.
+     */
+    @SerializedName("x5x") val x5x: String,
```

Comment:
the correct name of the field is **x5c** and it should contain **list of strings,** see the previous implementation

Reply:
correctness, is a relation between state & TIME I guess ;-)

Reply:
adjusted.